### DISCUSSION TITLE
Xdrip in app graph very small

### DISCUSSION THREAD

**Discussion by gregorgregor36** [NightscoutFoundation/xDrip#2664] [Q&A]
Hello, 

I wonder if someone can point me to a resolution.  My graph in xdrip+ sudenly chnaged to being very small.  This happened to me once before on a diffrent phone and I could not resolve it untill I started a new sensor.  When this sensor started is was fine but within 30 minutes mins it had changed.  I cannot find a way to change it back.  Anyone got any ideas or seen this before? Screnshot attached. 

Thanks in advance and thanks for all the hard work.  Xdrip has been a gamecharger for me!! 
Gregor

![Screenshot_20230208_125445_xDrip+](https://user-images.githubusercontent.com/76781151/217535639-685c7471-7706-4796-8ae7-d38ce950e7ca.jpg)

**Comment by Navid200** [comment]
Can you tap on what I have highlighted in your screenshot here?
![217535639-685c7471-7706-4796-8ae7-d38ce950e7ca](https://user-images.githubusercontent.com/51497406/217549994-356ee240-f06f-45d8-98bf-54e20163b4ac.jpg)

What happens if you tap on it?

**Comment by gregorgregor36** [comment]
Hi @Navid200 

Thanks for your reply. Screenshot attached.  I should say that I am using a NovoPen 6 and that looks like possibly when I nfc scanned the pen. 
![Screenshot_20230208_140057_xDrip+](https://user-images.githubusercontent.com/76781151/217552064-0482169d-ee90-48be-8f4b-b53acfcea2e4.jpg)

**Reply by Navid200** [reply]
There is an expected misbehavior by xDrip that could raise the max to values greater than 22.  This PR will fix that: https://github.com/NightscoutFoundation/xDrip/pull/2652

If you have a reading that is greater than 400 (22), xDrip expands the scale so that it can show the value on screen.
xDrip maintains that for 24 hours.
I intend to open another PR, after the one above is merged, that will limit the time penalty to 6 hours instead of 24.

**Reply by Navid200** [reply]
@gregorgregor36 You now have two screenshots of effectively the same issue.   Both of those screenshots are extremely large.
It makes it difficult for anyone to go through the thread.
You can scale down your images before attaching them.  You can see mine above.

But, would you please explain why you have attached the second screenshot?  Does it show anything that the first one did not?

**Comment by gregorgregor36** [comment]
Apologies, yes it shows the note at the bottom of the screenshot, this appeared when I clicked on were you asked me too. 

The note reads 

08:50 Priming 2.0U


* Also, thanks for your response and for poinitng me towards - #2652 

Much appreciated.

**Reply by Navid200** [reply]
Oh, I understand now!  Thanks.

I don't have experience with the pen.
The note shouldn't appear so far up on screen.  This could be a bug.
Let me see if I can find out who added that feature to inform them.

But, if you are willing to lose that note, you can tap on ADD NOTE and there may be an option to delete.   Deleting that may reduce the y axis max.  
But, even if it does, the max is not going to go below 22 as long as your 22 reading at around 10:30 is still on screen (tomorrow at 10:30).

**Reply by Navid200** [reply]
Would you please do a test?
Please go to NovoPen Insulin Pen and enable "Hide priming doses".

Does it affect the scale of your graph?
You may, or may not, need re restart xDrip.
